<!DOCTYPE html><html lang="en" class="__variable_e8ce0c __variable_5899e0"><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width, initial-scale=1"/><link rel="preload" href="/_next/static/media/739c2d8941231bb4-s.p.woff2" as="font" crossorigin="" type="font/woff2"/><link rel="preload" href="/_next/static/media/e4af272ccee01ff0-s.p.woff2" as="font" crossorigin="" type="font/woff2"/><link rel="preload" as="image" href="/images/team/rsg-logo.png"/><link rel="stylesheet" href="/_next/static/css/e1dee2bea2cc6267.css" data-precedence="next"/><link rel="stylesheet" href="/_next/static/css/c6b8421c7aae3eab.css" data-precedence="next"/><link rel="preload" as="script" fetchPriority="low" href="/_next/static/chunks/webpack-3181cac28b86fd42.js"/><script src="/_next/static/chunks/4bd1b696-a10ee7274c8b3d1f.js" async=""></script><script src="/_next/static/chunks/684-3d491d3500c15b6e.js" async=""></script><script src="/_next/static/chunks/main-app-c9d88178a7240503.js" async=""></script><script src="/_next/static/chunks/874-39484d5feff3b265.js" async=""></script><script src="/_next/static/chunks/app/layout-088455a57c4a5537.js" async=""></script><script src="/_next/static/chunks/63-9d11383f2d051bdd.js" async=""></script><script src="/_next/static/chunks/app/news/page-4bdc83f51cd4ad8a.js" async=""></script><script src="/_next/static/chunks/app/news/%5Bslug%5D/page-c3cccb97bb401f88.js" async=""></script><meta name="next-size-adjust" content=""/><link rel="apple-touch-icon" sizes="180x180" href="/favicon/apple-touch-icon.png"/><link rel="icon" type="image/png" sizes="32x32" href="/favicon/favicon-32x32.png"/><link rel="icon" type="image/png" sizes="16x16" href="/favicon/favicon-16x16.png"/><link rel="manifest" href="/favicon/site.webmanifest"/><link rel="mask-icon" href="/favicon/safari-pinned-tab.svg" color="#000000"/><link rel="shortcut icon" href="/favicon/favicon.ico"/><meta name="msapplication-TileColor" content="#000000"/><meta name="msapplication-config" content="/favicon/browserconfig.xml"/><meta name="theme-color" content="#000"/><link rel="alternate" type="application/rss+xml" href="/feed.xml"/><title>Announcing the second RSG Belgium Hackathon | RSG Belgium</title><meta property="og:title" content="Announcing the second RSG Belgium Hackathon"/><meta property="og:type" content="article"/><meta name="twitter:card" content="summary_large_image"/><meta name="twitter:title" content="Announcing the second RSG Belgium Hackathon"/><script>document.querySelectorAll('body link[rel="icon"], body link[rel="apple-touch-icon"]').forEach(el => document.head.appendChild(el))</script><script src="/_next/static/chunks/polyfills-42372ed130431b0a.js" noModule=""></script></head><body class="dark:bg-slate-900 dark:text-slate-400"><div hidden=""><!--$--><!--/$--></div><script>((e,t,r,n,o,a,i,u)=>{let l=document.documentElement,s=["light","dark"];function c(t){var r;(Array.isArray(e)?e:[e]).forEach(e=>{let r="class"===e,n=r&&a?o.map(e=>a[e]||e):o;r?(l.classList.remove(...n),l.classList.add(a&&a[t]?a[t]:t)):l.setAttribute(e,t)}),r=t,u&&s.includes(r)&&(l.style.colorScheme=r)}if(n)c(n);else try{let e=localStorage.getItem(t)||r,n=i&&"system"===e?window.matchMedia("(prefers-color-scheme: dark)").matches?"dark":"light":e;c(n)}catch(e){}})("class","theme","system",null,["light","dark"],null,true,true)</script><div class="min-h-screen flex flex-col"><header class="bg-white dark:bg-slate-900 shadow-md sticky top-0 z-50"><div class="max-w-6xl mx-auto px-4 sm:px-6 lg:px-8"><div class="flex items-center justify-between h-16"><div class="flex-shrink-0"><a class="text-2xl font-bold text-primary hover:text-secondary dark:text-lightblue dark:hover:text-white" href="/">RSG-Belgium</a></div><nav class="hidden md:flex md:items-center md:space-x-4"><a class="px-3 py-2 rounded-md text-sm font-medium text-secondary hover:bg-primary hover:text-white transition-colors dark:text-lightblue dark:hover:bg-secondary dark:hover:text-white" href="/about">About</a><a class="px-3 py-2 rounded-md text-sm font-medium text-secondary hover:bg-primary hover:text-white transition-colors dark:text-lightblue dark:hover:bg-secondary dark:hover:text-white" href="/news">News</a><a class="px-3 py-2 rounded-md text-sm font-medium text-secondary hover:bg-primary hover:text-white transition-colors dark:text-lightblue dark:hover:bg-secondary dark:hover:text-white" href="/organization">Organization</a><a class="px-3 py-2 rounded-md text-sm font-medium text-secondary hover:bg-primary hover:text-white transition-colors dark:text-lightblue dark:hover:bg-secondary dark:hover:text-white" href="/contact">Contact</a><a class="font-semibold rounded-md focus:outline-none focus:ring-2 focus:ring-offset-2 transition ease-in-out duration-150 px-3 py-1.5 text-sm button-primary dark:bg-secondary dark:text-lightblue dark:hover:text-lightblue " href="/slack">Join Slack</a><div style="width:70px;height:30px"></div> </nav><div class="md:hidden flex items-center"><button type="button" class="inline-flex items-center justify-center p-2 rounded-md text-gray-400 hover:text-gray-500 hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-inset focus:ring-blue-500" aria-controls="mobile-menu" aria-expanded="false"><span class="sr-only">Open main menu</span><svg class="block h-6 w-6" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor" aria-hidden="true"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16m-7 6h7"></path></svg></button></div></div></div></header><main class="flex-grow"> <main><div class="border-b dark:bg-slate-800 bg-neutral-50 border-neutral-200"><div class="container mx-auto px-5"><div class="py-2 text-center text-sm">Welcome to RSG-Belgium.<!-- --> <a href="https://github.com/rsg-belgium/website" class="underline hover:text-blue-600 duration-200 transition-colors">View source on GitHub</a>.</div></div></div><div class="container mx-auto px-5"><article class="mb-32"><h1 class="text-6xl md:text-7xl lg:text-8xl font-bold tracking-tighter leading-tight md:leading-none mb-12 text-center md:text-left">Announcing the second RSG Belgium Hackathon</h1><div class="hidden md:block md:mb-12"><div class="flex items-center"><img src="/images/team/rsg-logo.png" class="w-12 h-12 rounded-full mr-4" alt="RSG-Belgium"/><div class="text-xl font-bold">RSG-Belgium</div></div></div><div class="max-w-2xl mx-auto"><div class="block md:hidden mb-6"><div class="flex items-center"><img src="/images/team/rsg-logo.png" class="w-12 h-12 rounded-full mr-4" alt="RSG-Belgium"/><div class="text-xl font-bold">RSG-Belgium</div></div></div><div class="mb-6 text-lg"><time dateTime="2019-10-10 10:00">October 10, 2019</time></div></div><div class="max-w-2xl mx-auto"><div class="markdown-styles_markdown__ORfR8"><p>After last year's success, RSG Belgium is excited to announce the second RSG Belgium Hackathon! It will will take place in Brussels once again, on the 6th of December 2019.</p>
<p>Like last year, we want to provide an environment for young researchers to improve their problem solving skills and to cooperate in a multidisciplinary team with a common goal.</p>
<p>[<img src="/assets/img/poster/hackathon_2019.png" alt="full">{: .center}]({{ site.baseurl }}/assets/img/poster/hackathon_2019.png)</p>
<p>We have chosen a number of different projects that could be tackled during the event. We ask you to already indicate your preference during registration, so that we can make preparations for the most popular topics.</p>
<h2><a href="https://forms.gle/mcVQXsDNRxPD91Tf9" target="_blank" rel="noopener noreferrer">Register now!</a> - registration deadline: 10 November 2019</h2>
<h2>Where and when?</h2>
<ul>
<li>Date and time: 6/12/2019, 09:00 until 18:00</li>
<li>Place: Université Libre de Bruxelles, La Plaine Campus, Building NO, 9th floor, room: 2NO906 ("Salle des professeurs"), Brussels, Belgium</li>
</ul>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Registration time</td>
<td>8:30</td>
</tr>
<tr>
<td>Introduction and ice-breaker</td>
<td>9:15</td>
</tr>
<tr>
<td>Hackathon kick-off</td>
<td>9:45</td>
</tr>
<tr>
<td>Lunch</td>
<td>12:30</td>
</tr>
<tr>
<td>Project wrap-up and demonstration pitches</td>
<td>17:30</td>
</tr>
<tr>
<td>Social dinner</td>
<td>19:00</td>
</tr>
</tbody>
</table>
<p><strong>More information on how to get there is provided <a href="#how-to-get-there">below</a>.</strong></p>
<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d4363.244626829035!2d4.3951958277813965!3d50.82110086482844!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0xdce9335a43c9dfb5!2sBatiment+NO!5e0!3m2!1sen!2sbe!4v1538399705300" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
<h2>Projects</h2>
<h3>Predicting the diagnosis or progress of a patient based on other patient records</h3>
<p>As technology advances, a lot of medical institutes started using Electronic Medical Records (EMRs) to record a patient’s condition, including diagnostic information, illness progress and treatment results. EMRs has been recognised as a valuable resource for large-scale analysis for the development of several applications, including machine learning algorithms, decision support tools, and clinical research. In this project we will look at freely available hospital data (demos of databases) and we will try to find interesting patterns or develop predictive methods: by comparing a patient's symptoms with other patients who had similar symptoms in the past, we could predict or validate his diagnosis, how his condition will progress with time, what treatment suits the case best and even how this patient could respond to therapy.</p>
<h3>How much time will it take for your next paper to be reviewed?</h3>
<p>We all probably have felt it one way or the other, the frustration of sometimes having to wait months or years for our new exciting paper to be reviewed and, hopefully, accepted. What if maybe we would be able to save ourselves the anxiety and be able to at least predict the duration of our next paper review, so that we don't have to refresh the review page every 10 minutes? For this project, we will collect meta-data from published papers, such as journal info, h-indices of authors, submission and publication dates, keywords, etc. and create a predictor that can give an estimation of the review time. Does the review time depend on the impact factor of the journal? Has the review duration changed over the years? Are we able to provide review trends for different scientific fields?</p>
<h3>Predicting the functional category of gene families based on their properties</h3>
<p>Predicting the specific function of a protein is a huge challenge in biology because for many proteins there are no known homologs available with an experimentally determined function. This is especially the case for non-model organisms. But, proteins can also be divided into very broad functional categories (e.g. "Translation, ribosomal structure and biogenesis", "Energy production and conversion", ...), and predicting the functional category of a protein might be more feasible than predicting the specific function it performs. In this project, we will attempt to predict the functional category of proteins based on the properties of their sequence (e.g. sequence motifs), the protein itself (e.g. interaction partners) or even the entire protein family (e.g. copy number per genome, speed of evolution, ...).</p>
<h3>What is the Kardashian Index (K-Index) of your research?</h3>
<p>The BLAST paper has been one of the most influential papers in the history of bioinformatics, but surely didn't make a big splash in the media back then. Vice-versa, some labs are loud on social media, but produce only medium quality research. But does this actually mean anything ... academically speaking? By collecting various online articles and looking at their coverage by the media (Altmetric, shares, etc.), we can compare this coverage to the academic side of things: e.g. at the impact factor of papers, their citations. We can even compare the media coverage and the “robustness” of the research: does high media coverage correlate with more experiments in silico vs validated research, or even higher/lower p-values or particular subjects?</p>
<h3>How do network databases evolve with time?</h3>
<p>There are various types of molecular interaction data being used in bioinformatics, deposited in big databases that can be used to investigate networks. Unfortunately, these databases tend to grow over time and this might severely affect the properties of the networks. In this project we will collect historical versions of databases and describe their evolution through time in terms of their network properties (e.g. size, connectivity, the number of hubs, etc). Are we discovering more information about hubs through time? What types of networks do we discover? What happens with the evidence behind the data or their origin? Examples of datasets could be: protein-protein interaction networks, co-expression networks, transcription factors, etc.</p>
<h3>Developing your own Twitterbot</h3>
<p>Keeping track of the research or available datasets in your field is always a challenge. What if you had your personal Twitter bot to take care of this for you? In this project we will code a bot covering your topic of interest, in the LactoBot style (<a href="https://twitter.com/Lactobot" target="_blank" rel="noopener noreferrer">https://twitter.com/Lactobot</a>).</p>
<h2>How to get there?</h2>
<p>In general, the university is accessible by metro, bus and train. Specific signs for the hackathon will be available at the main bus and metro stops outside the university to guide you.</p>
<p>Please also consult the following map of the ULB Campus de la plaine to guide you to the NO Building.</p>
<p><img src="/assets/img/maps/hackathon-LaPlaineMap.png" alt="full"></p>
<h3>Access from the Bruxelles-Central train station</h3>
<p>From the Brussels Central station the university is accessible either by metro or bus.</p>
<ul>
<li>Metro: You follow the metro signs inside the station. You take the metro line 5 towards “Herrmann-Debroux” and you take off at the station “Delta”. There, you follow the exit with the “ULB” sign. The building NO is within 4 minutes walking distance. Estimated total time: 20 minutes.</li>
<li>Bus: You take the line 71, whose stop is just next to the main entrance of the train station, towards “Delta”. You take off at the stop “Fraiteur” and you pass at the other side of the road, towards the university Entrance 2. The building NO is within 2 minutes walking distance. Estimated total time: 30 minutes.</li>
</ul>
<h3>Access from the Bruxelles-Midi (Brussel-Zuid) train station</h3>
<p>You can follow the metro signs inside the Bruxelles-Midi station. You can take either the metro line 2 or line 6 towards “Elisabeth” and you stop at the metro station “Art-Loi (Kunst-Wet)”. There, you change for the metro line 5 towards “Herrmann-Debroux” and you take off at the station “Delta”. At the station you follow the exit with the “ULB” sign. The building NO is within 4 minutes walking distance.</p>
<p>Estimated total time: 25 minutes.</p>
<h3>Access from the train station Etterbeek (Etterbeek Gare)</h3>
<p>From the train station Etterbeek the university is reachable within walking distance. Walking out of the station you follow the Boulevard Général Jacques on the right and then you turn right at the Boulevard de la Plaine. After walking around 600 m, you can find the Access 2 of ULB on your left.</p>
<p>Estimated total time: 13 minutes.</p>
<h3>Access from the train station Delta</h3>
<p>At the station, you follow the exit with the “ULB” sign. The building NO is within 4 minutes walking distance.</p>
<h3>General access with bus or tram</h3>
<p>In general, the university is reachable with the bus line 71 (direction “Delta”, stop at “Fraiteur”), bus line 72 (direction “ULB”, stop at “Fraiteur”).</p>
<p>The closest tram stop is for the tram line 95 (direction “Wiener”, stop at “Thys”).</p>
<h3>General access with metro</h3>
<p>ULB is reachable with the metro line 5 (direction “Herrmann-Debroux”, metro station “Delta”). There, you follow the exit with the “ULB” sign. The building NO is within 4 minutes walking distance.</p>
</div></div></article></div></main><!--$--><!--/$--></main><footer class="bg-gray-800 text-gray-300 border-t border-gray-700"><div class="container mx-auto px-5"><div class="py-12 px-4 sm:px-6 lg:px-8"><div class="grid grid-cols-1 md:grid-cols-3 gap-8"><div class="mb-6 md:mb-0"><h3 class="text-lg font-semibold text-white mb-3">RSG-Belgium</h3><p class="text-sm leading-relaxed">Fostering a community of bioinformatics and computational biology students and young researchers in Belgium.</p></div><div class="mb-6 md:mb-0"><h3 class="text-lg font-semibold text-white mb-3">Quick Links</h3><ul class="space-y-2"><li><a class="text-sm text-lightblue hover:text-white hover:underline transition-colors" href="/about">About</a></li><li><a class="text-sm text-lightblue hover:text-white hover:underline transition-colors" href="/news">News</a></li><li><a class="text-sm text-lightblue hover:text-white hover:underline transition-colors" href="/organization">Organization</a></li><li><a class="text-sm text-lightblue hover:text-white hover:underline transition-colors" href="/contact">Contact</a></li><li><a class="text-sm text-lightblue hover:text-white hover:underline transition-colors" href="/slack">Join Slack</a></li></ul></div><div><h3 class="text-lg font-semibold text-white mb-3 dark:text-lightblue">Connect With Us</h3><div class="flex space-x-4 mb-4"><a href="https://twitter.com/rsgbelgium" target="_blank" rel="noopener noreferrer" class="text-lightblue hover:text-white transition-colors" aria-label="Twitter"><span class="text-xl">T</span></a><a href="https://www.facebook.com/RSGBelgium" target="_blank" rel="noopener noreferrer" class="text-lightblue hover:text-white transition-colors" aria-label="Facebook"><span class="text-xl">F</span></a><a href="https://www.linkedin.com/company/rsg-belgium/" target="_blank" rel="noopener noreferrer" class="text-lightblue hover:text-white transition-colors" aria-label="LinkedIn"><span class="text-xl">L</span></a></div><p class="text-sm text-lightblue">Email: <a href="mailto:iscb.rsg.belgium@gmail.com" class="hover:text-white hover:underline text-lightblue">iscb.rsg.belgium@gmail.com</a></p></div></div><div class="mt-10 pt-8 border-t border-gray-700 text-center"><p class="text-sm">© <!-- -->2025<!-- --> RSG-Belgium. All rights reserved.</p></div></div></div></footer></div><script src="/_next/static/chunks/webpack-3181cac28b86fd42.js" async=""></script><script>(self.__next_f=self.__next_f||[]).push([0])</script><script>self.__next_f.push([1,"1:\"$Sreact.fragment\"\n2:I[881,[\"874\",\"static/chunks/874-39484d5feff3b265.js\",\"177\",\"static/chunks/app/layout-088455a57c4a5537.js\"],\"ThemeProvider\"]\n3:I[6280,[\"874\",\"static/chunks/874-39484d5feff3b265.js\",\"177\",\"static/chunks/app/layout-088455a57c4a5537.js\"],\"default\"]\n4:I[7555,[],\"\"]\n5:I[1295,[],\"\"]\n6:I[6874,[\"874\",\"static/chunks/874-39484d5feff3b265.js\",\"63\",\"static/chunks/63-9d11383f2d051bdd.js\",\"324\",\"static/chunks/app/news/page-4bdc83f51cd4ad8a.js\"],\"\"]\n8:I[9665,[],\"OutletBoundary\"]\nb:I[4911,[],\"AsyncMetadataOutlet\"]\nd:I[9665,[],\"ViewportBoundary\"]\nf:I[9665,[],\"MetadataBoundary\"]\n11:I[6614,[],\"\"]\n:HL[\"/_next/static/media/739c2d8941231bb4-s.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/media/e4af272ccee01ff0-s.p.woff2\",\"font\",{\"crossOrigin\":\"\",\"type\":\"font/woff2\"}]\n:HL[\"/_next/static/css/e1dee2bea2cc6267.css\",\"style\"]\n:HL[\"/_next/static/css/c6b8421c7aae3eab.css\",\"style\"]\n"])</script><script>self.__next_f.push([1,"0:{\"P\":null,\"b\":\"GqvEW1v4UH9LbpXGzS2LC\",\"p\":\"\",\"c\":[\"\",\"news\",\"2019-10-10-second-hackathon\"],\"i\":false,\"f\":[[[\"\",{\"children\":[\"news\",{\"children\":[[\"slug\",\"2019-10-10-second-hackathon\",\"d\"],{\"children\":[\"__PAGE__\",{}]}]}]},\"$undefined\",\"$undefined\",true],[\"\",[\"$\",\"$1\",\"c\",{\"children\":[[[\"$\",\"link\",\"0\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/e1dee2bea2cc6267.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"$undefined\"}]],[\"$\",\"html\",null,{\"lang\":\"en\",\"className\":\"__variable_e8ce0c __variable_5899e0\",\"suppressHydrationWarning\":true,\"children\":[[\"$\",\"head\",null,{\"children\":[[\"$\",\"link\",null,{\"rel\":\"apple-touch-icon\",\"sizes\":\"180x180\",\"href\":\"/favicon/apple-touch-icon.png\"}],[\"$\",\"link\",null,{\"rel\":\"icon\",\"type\":\"image/png\",\"sizes\":\"32x32\",\"href\":\"/favicon/favicon-32x32.png\"}],[\"$\",\"link\",null,{\"rel\":\"icon\",\"type\":\"image/png\",\"sizes\":\"16x16\",\"href\":\"/favicon/favicon-16x16.png\"}],[\"$\",\"link\",null,{\"rel\":\"manifest\",\"href\":\"/favicon/site.webmanifest\"}],[\"$\",\"link\",null,{\"rel\":\"mask-icon\",\"href\":\"/favicon/safari-pinned-tab.svg\",\"color\":\"#000000\"}],[\"$\",\"link\",null,{\"rel\":\"shortcut icon\",\"href\":\"/favicon/favicon.ico\"}],[\"$\",\"meta\",null,{\"name\":\"msapplication-TileColor\",\"content\":\"#000000\"}],[\"$\",\"meta\",null,{\"name\":\"msapplication-config\",\"content\":\"/favicon/browserconfig.xml\"}],[\"$\",\"meta\",null,{\"name\":\"theme-color\",\"content\":\"#000\"}],[\"$\",\"link\",null,{\"rel\":\"alternate\",\"type\":\"application/rss+xml\",\"href\":\"/feed.xml\"}]]}],[\"$\",\"body\",null,{\"className\":\"dark:bg-slate-900 dark:text-slate-400\",\"children\":[\"$\",\"$L2\",null,{\"attribute\":\"class\",\"defaultTheme\":\"system\",\"enableSystem\":true,\"disableTransitionOnChange\":true,\"children\":[\"$\",\"div\",null,{\"className\":\"min-h-screen flex flex-col\",\"children\":[[\"$\",\"$L3\",null,{}],[\"$\",\"main\",null,{\"className\":\"flex-grow\",\"children\":[\" \",[\"$\",\"$L4\",null,{\"parallelRouterKey\":\"children\",\"error\":\"$undefined\",\"errorStyles\":\"$undefined\",\"errorScripts\":\"$undefined\",\"template\":[\"$\",\"$L5\",null,{}],\"templateStyles\":\"$undefined\",\"templateScripts\":\"$undefined\",\"notFound\":[[[\"$\",\"title\",null,{\"children\":\"404: This page could not be found.\"}],[\"$\",\"div\",null,{\"style\":{\"fontFamily\":\"system-ui,\\\"Segoe UI\\\",Roboto,Helvetica,Arial,sans-serif,\\\"Apple Color Emoji\\\",\\\"Segoe UI Emoji\\\"\",\"height\":\"100vh\",\"textAlign\":\"center\",\"display\":\"flex\",\"flexDirection\":\"column\",\"alignItems\":\"center\",\"justifyContent\":\"center\"},\"children\":[\"$\",\"div\",null,{\"children\":[[\"$\",\"style\",null,{\"dangerouslySetInnerHTML\":{\"__html\":\"body{color:#000;background:#fff;margin:0}.next-error-h1{border-right:1px solid rgba(0,0,0,.3)}@media (prefers-color-scheme:dark){body{color:#fff;background:#000}.next-error-h1{border-right:1px solid rgba(255,255,255,.3)}}\"}}],[\"$\",\"h1\",null,{\"className\":\"next-error-h1\",\"style\":{\"display\":\"inline-block\",\"margin\":\"0 20px 0 0\",\"padding\":\"0 23px 0 0\",\"fontSize\":24,\"fontWeight\":500,\"verticalAlign\":\"top\",\"lineHeight\":\"49px\"},\"children\":404}],[\"$\",\"div\",null,{\"style\":{\"display\":\"inline-block\"},\"children\":[\"$\",\"h2\",null,{\"style\":{\"fontSize\":14,\"fontWeight\":400,\"lineHeight\":\"49px\",\"margin\":0},\"children\":\"This page could not be found.\"}]}]]}]}]],[]],\"forbidden\":\"$undefined\",\"unauthorized\":\"$undefined\"}]]}],[\"$\",\"footer\",null,{\"className\":\"bg-gray-800 text-gray-300 border-t border-gray-700\",\"children\":[\"$\",\"div\",null,{\"className\":\"container mx-auto px-5\",\"children\":[\"$\",\"div\",null,{\"className\":\"py-12 px-4 sm:px-6 lg:px-8\",\"children\":[[\"$\",\"div\",null,{\"className\":\"grid grid-cols-1 md:grid-cols-3 gap-8\",\"children\":[[\"$\",\"div\",null,{\"className\":\"mb-6 md:mb-0\",\"children\":[[\"$\",\"h3\",null,{\"className\":\"text-lg font-semibold text-white mb-3\",\"children\":\"RSG-Belgium\"}],[\"$\",\"p\",null,{\"className\":\"text-sm leading-relaxed\",\"children\":\"Fostering a community of bioinformatics and computational biology students and young researchers in Belgium.\"}]]}],[\"$\",\"div\",null,{\"className\":\"mb-6 md:mb-0\",\"children\":[[\"$\",\"h3\",null,{\"className\":\"text-lg font-semibold text-white mb-3\",\"children\":\"Quick Links\"}],[\"$\",\"ul\",null,{\"className\":\"space-y-2\",\"children\":[[\"$\",\"li\",\"About\",{\"children\":[\"$\",\"$L6\",null,{\"href\":\"/about\",\"className\":\"text-sm text-lightblue hover:text-white hover:underline transition-colors\",\"children\":\"About\"}]}],[\"$\",\"li\",\"News\",{\"children\":[\"$\",\"$L6\",null,{\"href\":\"/news\",\"className\":\"text-sm text-lightblue hover:text-white hover:underline transition-colors\",\"children\":\"News\"}]}],[\"$\",\"li\",\"Organization\",{\"children\":[\"$\",\"$L6\",null,{\"href\":\"/organization\",\"className\":\"text-sm text-lightblue hover:text-white hover:underline transition-colors\",\"children\":\"Organization\"}]}],[\"$\",\"li\",\"Contact\",{\"children\":[\"$\",\"$L6\",null,{\"href\":\"/contact\",\"className\":\"text-sm text-lightblue hover:text-white hover:underline transition-colors\",\"children\":\"Contact\"}]}],[\"$\",\"li\",\"Join Slack\",{\"children\":[\"$\",\"$L6\",null,{\"href\":\"/slack\",\"className\":\"text-sm text-lightblue hover:text-white hover:underline transition-colors\",\"children\":\"Join Slack\"}]}]]}]]}],[\"$\",\"div\",null,{\"children\":[[\"$\",\"h3\",null,{\"className\":\"text-lg font-semibold text-white mb-3 dark:text-lightblue\",\"children\":\"Connect With Us\"}],[\"$\",\"div\",null,{\"className\":\"flex space-x-4 mb-4\",\"children\":[[\"$\",\"a\",\"Twitter\",{\"href\":\"https://twitter.com/rsgbelgium\",\"target\":\"_blank\",\"rel\":\"noopener noreferrer\",\"className\":\"text-lightblue hover:text-white transition-colors\",\"aria-label\":\"Twitter\",\"children\":[\"$\",\"span\",null,{\"className\":\"text-xl\",\"children\":\"T\"}]}],[\"$\",\"a\",\"Facebook\",{\"href\":\"https://www.facebook.com/RSGBelgium\",\"target\":\"_blank\",\"rel\":\"noopener noreferrer\",\"className\":\"text-lightblue hover:text-white transition-colors\",\"aria-label\":\"Facebook\",\"children\":[\"$\",\"span\",null,{\"className\":\"text-xl\",\"children\":\"F\"}]}],[\"$\",\"a\",\"LinkedIn\",{\"href\":\"https://www.linkedin.com/company/rsg-belgium/\",\"target\":\"_blank\",\"rel\":\"noopener noreferrer\",\"className\":\"text-lightblue hover:text-white transition-colors\",\"aria-label\":\"LinkedIn\",\"children\":[\"$\",\"span\",null,{\"className\":\"text-xl\",\"children\":\"L\"}]}]]}],[\"$\",\"p\",null,{\"className\":\"text-sm text-lightblue\",\"children\":[\"Email: \",[\"$\",\"a\",null,{\"href\":\"mailto:iscb.rsg.belgium@gmail.com\",\"className\":\"hover:text-white hover:underline text-lightblue\",\"children\":\"iscb.rsg.belgium@gmail.com\"}]]}]]}]]}],[\"$\",\"div\",null,{\"className\":\"mt-10 pt-8 border-t border-gray-700 text-center\",\"children\":[\"$\",\"p\",null,{\"className\":\"text-sm\",\"children\":[\"© \",2025,\" RSG-Belgium. All rights reserved.\"]}]}]]}]}]}]]}]}]}]]}]]}],{\"children\":[\"news\",[\"$\",\"$1\",\"c\",{\"children\":[null,[\"$\",\"$L4\",null,{\"parallelRouterKey\":\"children\",\"error\":\"$undefined\",\"errorStyles\":\"$undefined\",\"errorScripts\":\"$undefined\",\"template\":[\"$\",\"$L5\",null,{}],\"templateStyles\":\"$undefined\",\"templateScripts\":\"$undefined\",\"notFound\":\"$undefined\",\"forbidden\":\"$undefined\",\"unauthorized\":\"$undefined\"}]]}],{\"children\":[[\"slug\",\"2019-10-10-second-hackathon\",\"d\"],[\"$\",\"$1\",\"c\",{\"children\":[null,[\"$\",\"$L4\",null,{\"parallelRouterKey\":\"children\",\"error\":\"$undefined\",\"errorStyles\":\"$undefined\",\"errorScripts\":\"$undefined\",\"template\":[\"$\",\"$L5\",null,{}],\"templateStyles\":\"$undefined\",\"templateScripts\":\"$undefined\",\"notFound\":\"$undefined\",\"forbidden\":\"$undefined\",\"unauthorized\":\"$undefined\"}]]}],{\"children\":[\"__PAGE__\",[\"$\",\"$1\",\"c\",{\"children\":[\"$L7\",[[\"$\",\"link\",\"0\",{\"rel\":\"stylesheet\",\"href\":\"/_next/static/css/c6b8421c7aae3eab.css\",\"precedence\":\"next\",\"crossOrigin\":\"$undefined\",\"nonce\":\"$undefined\"}]],[\"$\",\"$L8\",null,{\"children\":[\"$L9\",\"$La\",[\"$\",\"$Lb\",null,{\"promise\":\"$@c\"}]]}]]}],{},null,false]},null,false]},null,false]},null,false],[\"$\",\"$1\",\"h\",{\"children\":[null,[\"$\",\"$1\",\"8k_RCjsiebSwU1yY1uyzAv\",{\"children\":[[\"$\",\"$Ld\",null,{\"children\":\"$Le\"}],[\"$\",\"meta\",null,{\"name\":\"next-size-adjust\",\"content\":\"\"}]]}],[\"$\",\"$Lf\",null,{\"children\":\"$L10\"}]]}],false]],\"m\":\"$undefined\",\"G\":[\"$11\",\"$undefined\"],\"s\":false,\"S\":true}\n"])</script><script>self.__next_f.push([1,"12:\"$Sreact.suspense\"\n13:I[4911,[],\"AsyncMetadata\"]\n10:[\"$\",\"div\",null,{\"hidden\":true,\"children\":[\"$\",\"$12\",null,{\"fallback\":null,\"children\":[\"$\",\"$L13\",null,{\"promise\":\"$@14\"}]}]}]\na:null\n"])</script><script>self.__next_f.push([1,"e:[[\"$\",\"meta\",\"0\",{\"charSet\":\"utf-8\"}],[\"$\",\"meta\",\"1\",{\"name\":\"viewport\",\"content\":\"width=device-width, initial-scale=1\"}]]\n9:null\n"])</script><script>self.__next_f.push([1,"15:I[8430,[\"874\",\"static/chunks/874-39484d5feff3b265.js\",\"63\",\"static/chunks/63-9d11383f2d051bdd.js\",\"120\",\"static/chunks/app/news/%5Bslug%5D/page-c3cccb97bb401f88.js\"],\"default\"]\n16:T25bf,"])</script><script>self.__next_f.push([1,"\u003cp\u003eAfter last year's success, RSG Belgium is excited to announce the second RSG Belgium Hackathon! It will will take place in Brussels once again, on the 6th of December 2019.\u003c/p\u003e\n\u003cp\u003eLike last year, we want to provide an environment for young researchers to improve their problem solving skills and to cooperate in a multidisciplinary team with a common goal.\u003c/p\u003e\n\u003cp\u003e[\u003cimg src=\"/assets/img/poster/hackathon_2019.png\" alt=\"full\"\u003e{: .center}]({{ site.baseurl }}/assets/img/poster/hackathon_2019.png)\u003c/p\u003e\n\u003cp\u003eWe have chosen a number of different projects that could be tackled during the event. We ask you to already indicate your preference during registration, so that we can make preparations for the most popular topics.\u003c/p\u003e\n\u003ch2\u003e\u003ca href=\"https://forms.gle/mcVQXsDNRxPD91Tf9\" target=\"_blank\" rel=\"noopener noreferrer\"\u003eRegister now!\u003c/a\u003e - registration deadline: 10 November 2019\u003c/h2\u003e\n\u003ch2\u003eWhere and when?\u003c/h2\u003e\n\u003cul\u003e\n\u003cli\u003eDate and time: 6/12/2019, 09:00 until 18:00\u003c/li\u003e\n\u003cli\u003ePlace: Université Libre de Bruxelles, La Plaine Campus, Building NO, 9th floor, room: 2NO906 (\"Salle des professeurs\"), Brussels, Belgium\u003c/li\u003e\n\u003c/ul\u003e\n\u003ctable\u003e\n\u003cthead\u003e\n\u003ctr\u003e\n\u003cth\u003e\u003c/th\u003e\n\u003cth\u003e\u003c/th\u003e\n\u003c/tr\u003e\n\u003c/thead\u003e\n\u003ctbody\u003e\n\u003ctr\u003e\n\u003ctd\u003eRegistration time\u003c/td\u003e\n\u003ctd\u003e8:30\u003c/td\u003e\n\u003c/tr\u003e\n\u003ctr\u003e\n\u003ctd\u003eIntroduction and ice-breaker\u003c/td\u003e\n\u003ctd\u003e9:15\u003c/td\u003e\n\u003c/tr\u003e\n\u003ctr\u003e\n\u003ctd\u003eHackathon kick-off\u003c/td\u003e\n\u003ctd\u003e9:45\u003c/td\u003e\n\u003c/tr\u003e\n\u003ctr\u003e\n\u003ctd\u003eLunch\u003c/td\u003e\n\u003ctd\u003e12:30\u003c/td\u003e\n\u003c/tr\u003e\n\u003ctr\u003e\n\u003ctd\u003eProject wrap-up and demonstration pitches\u003c/td\u003e\n\u003ctd\u003e17:30\u003c/td\u003e\n\u003c/tr\u003e\n\u003ctr\u003e\n\u003ctd\u003eSocial dinner\u003c/td\u003e\n\u003ctd\u003e19:00\u003c/td\u003e\n\u003c/tr\u003e\n\u003c/tbody\u003e\n\u003c/table\u003e\n\u003cp\u003e\u003cstrong\u003eMore information on how to get there is provided \u003ca href=\"#how-to-get-there\"\u003ebelow\u003c/a\u003e.\u003c/strong\u003e\u003c/p\u003e\n\u003ciframe src=\"https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d4363.244626829035!2d4.3951958277813965!3d50.82110086482844!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0xdce9335a43c9dfb5!2sBatiment+NO!5e0!3m2!1sen!2sbe!4v1538399705300\" width=\"600\" height=\"450\" frameborder=\"0\" style=\"border:0\" allowfullscreen\u003e\u003c/iframe\u003e\n\u003ch2\u003eProjects\u003c/h2\u003e\n\u003ch3\u003ePredicting the diagnosis or progress of a patient based on other patient records\u003c/h3\u003e\n\u003cp\u003eAs technology advances, a lot of medical institutes started using Electronic Medical Records (EMRs) to record a patient’s condition, including diagnostic information, illness progress and treatment results. EMRs has been recognised as a valuable resource for large-scale analysis for the development of several applications, including machine learning algorithms, decision support tools, and clinical research. In this project we will look at freely available hospital data (demos of databases) and we will try to find interesting patterns or develop predictive methods: by comparing a patient's symptoms with other patients who had similar symptoms in the past, we could predict or validate his diagnosis, how his condition will progress with time, what treatment suits the case best and even how this patient could respond to therapy.\u003c/p\u003e\n\u003ch3\u003eHow much time will it take for your next paper to be reviewed?\u003c/h3\u003e\n\u003cp\u003eWe all probably have felt it one way or the other, the frustration of sometimes having to wait months or years for our new exciting paper to be reviewed and, hopefully, accepted. What if maybe we would be able to save ourselves the anxiety and be able to at least predict the duration of our next paper review, so that we don't have to refresh the review page every 10 minutes? For this project, we will collect meta-data from published papers, such as journal info, h-indices of authors, submission and publication dates, keywords, etc. and create a predictor that can give an estimation of the review time. Does the review time depend on the impact factor of the journal? Has the review duration changed over the years? Are we able to provide review trends for different scientific fields?\u003c/p\u003e\n\u003ch3\u003ePredicting the functional category of gene families based on their properties\u003c/h3\u003e\n\u003cp\u003ePredicting the specific function of a protein is a huge challenge in biology because for many proteins there are no known homologs available with an experimentally determined function. This is especially the case for non-model organisms. But, proteins can also be divided into very broad functional categories (e.g. \"Translation, ribosomal structure and biogenesis\", \"Energy production and conversion\", ...), and predicting the functional category of a protein might be more feasible than predicting the specific function it performs. In this project, we will attempt to predict the functional category of proteins based on the properties of their sequence (e.g. sequence motifs), the protein itself (e.g. interaction partners) or even the entire protein family (e.g. copy number per genome, speed of evolution, ...).\u003c/p\u003e\n\u003ch3\u003eWhat is the Kardashian Index (K-Index) of your research?\u003c/h3\u003e\n\u003cp\u003eThe BLAST paper has been one of the most influential papers in the history of bioinformatics, but surely didn't make a big splash in the media back then. Vice-versa, some labs are loud on social media, but produce only medium quality research. But does this actually mean anything ... academically speaking? By collecting various online articles and looking at their coverage by the media (Altmetric, shares, etc.), we can compare this coverage to the academic side of things: e.g. at the impact factor of papers, their citations. We can even compare the media coverage and the “robustness” of the research: does high media coverage correlate with more experiments in silico vs validated research, or even higher/lower p-values or particular subjects?\u003c/p\u003e\n\u003ch3\u003eHow do network databases evolve with time?\u003c/h3\u003e\n\u003cp\u003eThere are various types of molecular interaction data being used in bioinformatics, deposited in big databases that can be used to investigate networks. Unfortunately, these databases tend to grow over time and this might severely affect the properties of the networks. In this project we will collect historical versions of databases and describe their evolution through time in terms of their network properties (e.g. size, connectivity, the number of hubs, etc). Are we discovering more information about hubs through time? What types of networks do we discover? What happens with the evidence behind the data or their origin? Examples of datasets could be: protein-protein interaction networks, co-expression networks, transcription factors, etc.\u003c/p\u003e\n\u003ch3\u003eDeveloping your own Twitterbot\u003c/h3\u003e\n\u003cp\u003eKeeping track of the research or available datasets in your field is always a challenge. What if you had your personal Twitter bot to take care of this for you? In this project we will code a bot covering your topic of interest, in the LactoBot style (\u003ca href=\"https://twitter.com/Lactobot\" target=\"_blank\" rel=\"noopener noreferrer\"\u003ehttps://twitter.com/Lactobot\u003c/a\u003e).\u003c/p\u003e\n\u003ch2\u003eHow to get there?\u003c/h2\u003e\n\u003cp\u003eIn general, the university is accessible by metro, bus and train. Specific signs for the hackathon will be available at the main bus and metro stops outside the university to guide you.\u003c/p\u003e\n\u003cp\u003ePlease also consult the following map of the ULB Campus de la plaine to guide you to the NO Building.\u003c/p\u003e\n\u003cp\u003e\u003cimg src=\"/assets/img/maps/hackathon-LaPlaineMap.png\" alt=\"full\"\u003e\u003c/p\u003e\n\u003ch3\u003eAccess from the Bruxelles-Central train station\u003c/h3\u003e\n\u003cp\u003eFrom the Brussels Central station the university is accessible either by metro or bus.\u003c/p\u003e\n\u003cul\u003e\n\u003cli\u003eMetro: You follow the metro signs inside the station. You take the metro line 5 towards “Herrmann-Debroux” and you take off at the station “Delta”. There, you follow the exit with the “ULB” sign. The building NO is within 4 minutes walking distance. Estimated total time: 20 minutes.\u003c/li\u003e\n\u003cli\u003eBus: You take the line 71, whose stop is just next to the main entrance of the train station, towards “Delta”. You take off at the stop “Fraiteur” and you pass at the other side of the road, towards the university Entrance 2. The building NO is within 2 minutes walking distance. Estimated total time: 30 minutes.\u003c/li\u003e\n\u003c/ul\u003e\n\u003ch3\u003eAccess from the Bruxelles-Midi (Brussel-Zuid) train station\u003c/h3\u003e\n\u003cp\u003eYou can follow the metro signs inside the Bruxelles-Midi station. You can take either the metro line 2 or line 6 towards “Elisabeth” and you stop at the metro station “Art-Loi (Kunst-Wet)”. There, you change for the metro line 5 towards “Herrmann-Debroux” and you take off at the station “Delta”. At the station you follow the exit with the “ULB” sign. The building NO is within 4 minutes walking distance.\u003c/p\u003e\n\u003cp\u003eEstimated total time: 25 minutes.\u003c/p\u003e\n\u003ch3\u003eAccess from the train station Etterbeek (Etterbeek Gare)\u003c/h3\u003e\n\u003cp\u003eFrom the train station Etterbeek the university is reachable within walking distance. Walking out of the station you follow the Boulevard Général Jacques on the right and then you turn right at the Boulevard de la Plaine. After walking around 600 m, you can find the Access 2 of ULB on your left.\u003c/p\u003e\n\u003cp\u003eEstimated total time: 13 minutes.\u003c/p\u003e\n\u003ch3\u003eAccess from the train station Delta\u003c/h3\u003e\n\u003cp\u003eAt the station, you follow the exit with the “ULB” sign. The building NO is within 4 minutes walking distance.\u003c/p\u003e\n\u003ch3\u003eGeneral access with bus or tram\u003c/h3\u003e\n\u003cp\u003eIn general, the university is reachable with the bus line 71 (direction “Delta”, stop at “Fraiteur”), bus line 72 (direction “ULB”, stop at “Fraiteur”).\u003c/p\u003e\n\u003cp\u003eThe closest tram stop is for the tram line 95 (direction “Wiener”, stop at “Thys”).\u003c/p\u003e\n\u003ch3\u003eGeneral access with metro\u003c/h3\u003e\n\u003cp\u003eULB is reachable with the metro line 5 (direction “Herrmann-Debroux”, metro station “Delta”). There, you follow the exit with the “ULB” sign. The building NO is within 4 minutes walking distance.\u003c/p\u003e\n"])</script><script>self.__next_f.push([1,"7:[\"$\",\"main\",null,{\"children\":[[\"$\",\"div\",null,{\"className\":\"border-b dark:bg-slate-800 bg-neutral-50 border-neutral-200\",\"children\":[\"$\",\"div\",null,{\"className\":\"container mx-auto px-5\",\"children\":[\"$\",\"div\",null,{\"className\":\"py-2 text-center text-sm\",\"children\":[\"Welcome to RSG-Belgium.\",\" \",[\"$\",\"a\",null,{\"href\":\"https://github.com/rsg-belgium/website\",\"className\":\"underline hover:text-blue-600 duration-200 transition-colors\",\"children\":\"View source on GitHub\"}],\".\"]}]}]}],[\"$\",\"div\",null,{\"className\":\"container mx-auto px-5\",\"children\":[\"$\",\"article\",null,{\"className\":\"mb-32\",\"children\":[[\"$\",\"$L15\",null,{\"title\":\"Announcing the second RSG Belgium Hackathon\",\"coverImage\":\"$undefined\",\"date\":\"2019-10-10 10:00\",\"author\":{\"name\":\"RSG-Belgium\",\"picture\":\"/images/team/rsg-logo.png\"}}],[\"$\",\"div\",null,{\"className\":\"max-w-2xl mx-auto\",\"children\":[\"$\",\"div\",null,{\"className\":\"markdown-styles_markdown__ORfR8\",\"dangerouslySetInnerHTML\":{\"__html\":\"$16\"}}]}]]}]}]]}]\nc:{\"metadata\":[[\"$\",\"title\",\"0\",{\"children\":\"Announcing the second RSG Belgium Hackathon | RSG Belgium\"}],[\"$\",\"meta\",\"1\",{\"property\":\"og:title\",\"content\":\"Announcing the second RSG Belgium Hackathon\"}],[\"$\",\"meta\",\"2\",{\"property\":\"og:type\",\"content\":\"article\"}],[\"$\",\"meta\",\"3\",{\"name\":\"twitter:card\",\"content\":\"summary_large_image\"}],[\"$\",\"meta\",\"4\",{\"name\":\"twitter:title\",\"content\":\"Announcing the second RSG Belgium Hackathon\"}]],\"error\":null,\"digest\":\"$undefined\"}\n14:{\"metadata\":\"$c:metadata\",\"error\":null,\"digest\":\"$undefined\"}\n"])</script></body></html>