---
layout: single
title:  "Announcing the second RSG Belgium Hackathon"
date:   2019-10-10 10:00
categories: news event hackathon
toc: true
category: 'News'
---

After last year's success, RSG Belgium is excited to announce the second RSG Belgium Hackathon! It will will take place in Brussels once again, on the 6th of December 2019.

Like last year, we want to provide an environment for young researchers to improve their problem solving skills and to cooperate in a multidisciplinary team with a common goal.

[![full](/assets/img/poster/hackathon_2019.png){: .center}]({{ site.baseurl }}/assets/img/poster/hackathon_2019.png)

We have chosen a number of different projects that could be tackled during the event. We ask you to already indicate your preference during registration, so that we can make preparations for the most popular topics.

## [Register now!](https://forms.gle/mcVQXsDNRxPD91Tf9) - registration deadline: 10 November 2019

## Where and when?

- Date and time: 6/12/2019, 09:00 until 18:00
- Place: Université Libre de Bruxelles, La Plaine Campus, Building NO, 9th floor, room: 2NO906 ("Salle des professeurs"), Brussels, Belgium

|   |   |
|---|---|
|Registration time|8:30|
|Introduction and ice-breaker|9:15|
|Hackathon kick-off|9:45|
|Lunch|12:30|
|Project wrap-up and demonstration pitches|17:30|
|Social dinner|19:00|

**More information on how to get there is provided [below](#how-to-get-there).**

<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d4363.244626829035!2d4.3951958277813965!3d50.82110086482844!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0xdce9335a43c9dfb5!2sBatiment+NO!5e0!3m2!1sen!2sbe!4v1538399705300" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>

## Projects

### Predicting the diagnosis or progress of a patient based on other patient records

As technology advances, a lot of medical institutes started using Electronic Medical Records (EMRs) to record a patient’s condition, including diagnostic information, illness progress and treatment results. EMRs has been recognised as a valuable resource for large-scale analysis for the development of several applications, including machine learning algorithms, decision support tools, and clinical research. In this project we will look at freely available hospital data (demos of databases) and we will try to find interesting patterns or develop predictive methods: by comparing a patient's symptoms with other patients who had similar symptoms in the past, we could predict or validate his diagnosis, how his condition will progress with time, what treatment suits the case best and even how this patient could respond to therapy.

### How much time will it take for your next paper to be reviewed?

We all probably have felt it one way or the other, the frustration of sometimes having to wait months or years for our new exciting paper to be reviewed and, hopefully, accepted. What if maybe we would be able to save ourselves the anxiety and be able to at least predict the duration of our next paper review, so that we don't have to refresh the review page every 10 minutes? For this project, we will collect meta-data from published papers, such as journal info, h-indices of authors, submission and publication dates, keywords, etc. and create a predictor that can give an estimation of the review time. Does the review time depend on the impact factor of the journal? Has the review duration changed over the years? Are we able to provide review trends for different scientific fields?

### Predicting the functional category of gene families based on their properties

Predicting the specific function of a protein is a huge challenge in biology because for many proteins there are no known homologs available with an experimentally determined function. This is especially the case for non-model organisms. But, proteins can also be divided into very broad functional categories (e.g. "Translation, ribosomal structure and biogenesis", "Energy production and conversion", ...), and predicting the functional category of a protein might be more feasible than predicting the specific function it performs. In this project, we will attempt to predict the functional category of proteins based on the properties of their sequence (e.g. sequence motifs), the protein itself (e.g. interaction partners) or even the entire protein family (e.g. copy number per genome, speed of evolution, ...).

### What is the Kardashian Index (K-Index) of your research?

The BLAST paper has been one of the most influential papers in the history of bioinformatics, but surely didn't make a big splash in the media back then. Vice-versa, some labs are loud on social media, but produce only medium quality research. But does this actually mean anything ... academically speaking? By collecting various online articles and looking at their coverage by the media (Altmetric, shares, etc.), we can compare this coverage to the academic side of things: e.g. at the impact factor of papers, their citations. We can even compare the media coverage and the “robustness” of the research: does high media coverage correlate with more experiments in silico vs validated research, or even higher/lower p-values or particular subjects?

### How do network databases evolve with time?

There are various types of molecular interaction data being used in bioinformatics, deposited in big databases that can be used to investigate networks. Unfortunately, these databases tend to grow over time and this might severely affect the properties of the networks. In this project we will collect historical versions of databases and describe their evolution through time in terms of their network properties (e.g. size, connectivity, the number of hubs, etc). Are we discovering more information about hubs through time? What types of networks do we discover? What happens with the evidence behind the data or their origin? Examples of datasets could be: protein-protein interaction networks, co-expression networks, transcription factors, etc.

### Developing your own Twitterbot

Keeping track of the research or available datasets in your field is always a challenge. What if you had your personal Twitter bot to take care of this for you? In this project we will code a bot covering your topic of interest, in the LactoBot style (https://twitter.com/Lactobot).

## How to get there?

In general, the university is accessible by metro, bus and train. Specific signs for the hackathon will be available at the main bus and metro stops outside the university to guide you.

Please also consult the following map of the ULB Campus de la plaine to guide you to the NO Building.

![full](/assets/img/maps/hackathon-LaPlaineMap.png)

### Access from the Bruxelles-Central train station

From the Brussels Central station the university is accessible either by metro or bus.

- Metro: You follow the metro signs inside the station. You take the metro line 5 towards “Herrmann-Debroux” and you take off at the station “Delta”. There, you follow the exit with the “ULB” sign. The building NO is within 4 minutes walking distance. Estimated total time: 20 minutes.
- Bus: You take the line 71, whose stop is just next to the main entrance of the train station, towards “Delta”. You take off at the stop “Fraiteur” and you pass at the other side of the road, towards the university Entrance 2. The building NO is within 2 minutes walking distance. Estimated total time: 30 minutes.

### Access from the Bruxelles-Midi (Brussel-Zuid) train station

You can follow the metro signs inside the Bruxelles-Midi station. You can take either the metro line 2 or line 6 towards “Elisabeth” and you stop at the metro station “Art-Loi (Kunst-Wet)”. There, you change for the metro line 5 towards “Herrmann-Debroux” and you take off at the station “Delta”. At the station you follow the exit with the “ULB” sign. The building NO is within 4 minutes walking distance.

Estimated total time: 25 minutes.

### Access from the train station Etterbeek (Etterbeek Gare)

From the train station Etterbeek the university is reachable within walking distance. Walking out of the station you follow the Boulevard Général Jacques on the right and then you turn right at the Boulevard de la Plaine. After walking around 600 m, you can find the Access 2 of ULB on your left.

Estimated total time: 13 minutes.

### Access from the train station Delta

At the station, you follow the exit with the “ULB” sign. The building NO is within 4 minutes walking distance.

### General access with bus or tram

In general, the university is reachable with the bus line 71 (direction “Delta”, stop at “Fraiteur”), bus line 72 (direction “ULB”, stop at “Fraiteur”).

The closest tram stop is for the tram line 95 (direction “Wiener”, stop at “Thys”).

### General access with metro

ULB is reachable with the metro line 5 (direction “Herrmann-Debroux”, metro station “Delta”). There, you follow the exit with the “ULB” sign. The building NO is within 4 minutes walking distance.
